test/e2e/_move.js

```diff
@@ -27,24 +20,19 @@ import UpdateMovePage from './pages/update-move'
  * @param {object} [options.moveOptions] - config for move creation
  * See createMoveFixture for more details
  *
- * @returns {undefined} Creates person and move, checking that move has all expected values
+ * @returns {object} - created move object containing person data returned from createMoveFixture
  */
 export async function createMove(options = {}) {
-  await t.useRole(options.user)
-
   if (
     !options.moveOverrides ||
     (options.moveOverrides && !options.moveOverrides.from_location)
   ) {
-    await t.navigateTo(home)
-
-    const currentLocation = await page.nodes.locationMeta.getAttribute(
+    const currentLocationId = await page.nodes.locationMeta.getAttribute(
```

Comment:
Is it an idea to blow up if the meta location element doesn't have a value?

Reply:
Good point. What do you think is the best way? Should we throw an error? Or do you think we could get away with a TestCafe assertion to check that it's not undefined?